manifests/0000_70_cluster-network-operator_02_rbac.yaml

```diff
@@ -1,16 +1,42 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cluster-network-operator
+  namespace: openshift-network-operator
+automountServiceAccountToken: false
+
+---
+
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: default-account-cluster-network-operator
+  name: cluster-network-operator
   annotations:
```

Comment:
dug into https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-network-operator/1808/pull-ci-openshift-cluster-network-operator-master-e2e-gcp-ovn/1661034270396780544 on gmeet

I suspect you're missing these profile annotations on your serviceaccount which the CVO requires before creating the SA manifest.  And lack of the SA prevents the kube-controller-manager from being able to create a pod for the CNO replicaset.

Reply:
Thanks @deads2k for your help debugging this one!!